loan-payment-skip/loan-payment-skip-json-contract.md

```diff
@@ -80,4 +80,22 @@ Any and all errors should be conveyed via the following structure:
   "loggingErrorMessage":"if error processing, something to log"
 }
 ```
+### Response Messages & Corresponding Logging Error Messages
 
+*All Errors beging with 'We're sorry we've encountered an error. Please contact your credit union.' followed by a Logging Error Message.
+
+| Logging Error Message                                                                | Additional Notes As Needed |
```

Comment:
The "...As Needed" language is a little weird for me.  Maybe there was a reason it was added?